### PR TITLE
New intake code.

### PR DESCRIPTION
**Pull request by Beckoe**
New PR because the other is closed.

### CODE REVIEW

**Comment by Beckoe** [comment]
Working on the conflicts

File: RobotCode/src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 1]
```diff
@@ -88,11 +84,13 @@ public void periodic()
                 intakeRollerPidController.setReference(desiredVelocity, ControlType.kVelocity, 0);
         }
 
+        /** Ticks */
         public void setDesiredRotation(double desiredRotation)
         {
                 this.desiredRotation = desiredRotation;
         }
 
+        /** Ticks per second */
```
Comment (renatodellosso) [inline RobotCode/src/main/java/frc/robot/subsystems/IntakeSubsystem.java:93]: do `/** @param desiredVelocity Ticks per second */`

File: RobotCode/src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 2]
```diff
@@ -88,11 +84,13 @@ public void periodic()
                 intakeRollerPidController.setReference(desiredVelocity, ControlType.kVelocity, 0);
         }
 
+        /** Ticks */
```
Comment (renatodellosso) [inline RobotCode/src/main/java/frc/robot/subsystems/IntakeSubsystem.java:87]: Do `/** @param desiredRotation Ticks */`

Comment (renatodellosso) [reply RobotCode/src/main/java/frc/robot/subsystems/IntakeSubsystem.java:87]: Also specify what range this should be in

**Review by renatodellosso — CHANGES_REQUESTED** [review]
If making a Javadoc note about a parameter, you can do `/** @param paramName note */`. Also, specify what range rotations should be in.

File: RobotCode/src/main/java/frc/robot/subsystems/IntakeSubsystem.java (hunk 2 above)
Comment (Beckoe) [reply RobotCode/src/main/java/frc/robot/subsystems/IntakeSubsystem.java:87]: The range is currently unknown and will be known by constants as there will be a constant for the minimum (retracted) position and maximum (deployed) position.

Comment (renatodellosso) [reply RobotCode/src/main/java/frc/robot/subsystems/IntakeSubsystem.java]: Ok, that's fine

**Review by renatodellosso — APPROVED** [review]
Looks good

**Pull request merged**